lib/postageapp/failed_request.rb

```diff
@@ -39,6 +39,8 @@ def self.resend_all
         uid: filename
       ).send(true)
 
+      PostageApp.logger.info(receipt_response)
+
       if (receipt_response.fail?)
         return
       elsif (receipt_response.ok?)
```

Comment:
I think we can remove the `.fail?` and `.ok?` cases here. They'll both be handled just the same on line 63 since `ok` is not retryable and `fail` is, right?